jekyll/_cci2/creating-orbs.md

```diff
@@ -0,0 +1,332 @@
+    jobs:
+      my_inline_job:
+        parameters:
+          greeting_name:
+            type: string
+            default: olleh
+        executor: my_inline_executor
+        steps:
+          - my_inline_command:
+              name: <<parameters.greeting_name>>
+    commands:
+      my_inline_command:
+        parameters:
+          name:
+            type: string
+        steps:
+          - run: echo "hello <<parameters.name>>, from the inline command"
+    executors:
+      my_inline_executor:
+        parameters:
+          version:
+            type: string
+            default: "2.4"
+        docker:
+          - image: circleci/ruby:<<parameters.version>>
+
+workflows:
+  build-test-deploy:
+    jobs:
+      - inline_example/my_inline_job:
+          name: build
+      - inline_example/my_inline_job:
+          name: build2
+          greeting_name: world
+```
+
+## Publishing an Orb
+
+This section covers the tooling and flow of authoring and publishing your own orbs to the orb registry.
+
+Orbs may be authored inline in your config.yml file or authored separately and then published to to the orb registry for reuse across projects.
+
+[WARNING] Orbs are always world-readable. All published orbs can be read and used by anyone. They are not limited to just the members of your organization. In general, CircleCI strongly recommends that you do not put secrets or other sensitive variables into your configuration. Instead, use contexts or project environment variables and reference the names of those environment variables in your orbs.
+
+### Prerequisites
+
+Before publishing an orb, you will need to first opt-in to the new 3rd-Party Software terms and turn on orb publishing for your organization.
+
+<aside class="notice">
+Only an organization administrator can opt-in to the 3rd party Software terms. The organization admin will need to navigate to the organization Settings tab and complete the form on the Security page.
+</aside>
+
+### Orb Publishing Concepts
+
+The sections below describe important concepts that you should understand before publishing an orb.
+
+#### Orb Registry
+
+Each installation of CircleCI has a single orb registry. The registry on circleci.com serves as the master source for all certified namespaces and orbs and is the only orb registry that users of circleci.com can use.
+
+When orb features come to the server installation of CircleCI, each installation will have its own registry that operators can control.
+
+<aside class="notice">
+Currently, orb features are not yet supported on server installations.
+</aside>
+
+#### Namespaces
+
+Namespaces are used to organize a set of orbs. Each namespace has a unique and immutable name within the registry, and each orb in a namespace has a unique name. For example, the orb ```circleci/rails``` means the rails orb in the ```circleci``` namespace may coexist in the registry with an orb called ```somenamespace/rails``` because they are in separate namespaces.
```

Comment:
The fully-qualified orb name is a bit different from just the orb name. This paragraph is correct as-is, the distinction between and orb name and a namespace name could be a bit more clear, like:

> means the ```rails``` in the ```circleci``` namespace may coexist in the registry with an orb called ```rails``` in the ```somenamespace``` namespace (the ```somenamespace/rails``` orb) because they are in separate namespaces.

Reply:
thanks, I simplified this to make it more readable: For example, the `circleci/rails` orb may coexist in the registry with an orb called ```hannah/rails``` because they are in separate namespaces.